docs/docs/default-actions.mdx

```diff
@@ -187,3 +187,39 @@ Overriding the default action for forms should **only** be used during the proce
 migrating from Rasa Open Source 1 to 2. In this case you can override the default
 action to instruct Rasa Open Source to use the deprecated `FormAction` which is part of
 the Rasa SDK.
+
+## action_unlikely_intent
+
+Rasa Open Source triggers `action_unlikely_intent` via [`IntentTEDPolicy`](./policies.mdx#intenttedpolicy).
+You can control how often this action is predicted by tuning the [`tolerance`](./policies.mdx#intenttedpolicy)
+parameter of `IntentTEDPolicy`.
+
+### Customization
+
+You can customize your assistant's behaviour to configure what should happen once `action_unlikely_intent`
+is triggered. For example, as a follow up you can trigger a hand-off to a human agent with a rule:
+
+```yaml
+- rule: trigger human handoff with action_unlikely_intent
+  steps:
+    - action: action_unlikely_intent
+    - action: ask_human_handoff
+    - intent: affirm
+    - action: trigger_human_handoff
+```
+
+Alternatively, you can also override it's behaviour as a `custom action`[./custom-actions.mdx]:
+
+```python
+class ActionUnlikelyIntent:
+
+    def name(self) -> Text:
+        return "action_unlikely_intent"
+
+    async def run(
+        self, dispatcher, tracker: Tracker, domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+
+        # Implement custom logic here
+        return []
+```
```

Comment:
@JEM-Mosig You had tested this right?

Reply:
Yes, it did work with the super airline bot. But I had to add the action name to the domain.

Reply:
@alwx This is related to the [slack thread](https://rasa-hq.slack.com/archives/C01HHMR4X8S/p1622624011188400?thread_ts=1622620786.185400&cid=C01HHMR4X8S)